openfisca_core/holders.py

```diff
@@ -100,7 +100,7 @@ def calculate(self, period, **parameters):
     def calculate_output(self, period):
         return self.formula.calculate_output(period)
 
-    def clone(self):
+    def clone(self, new_entity):
```

Comment:
Rename `new_entity` to `entity` ? `new_entity` name assumes something out of the bounds of this method.